src/aof.c

```diff
@@ -717,18 +1197,12 @@ int loadAppendOnlyFile(char *filename) {
             serverLog(LL_WARNING,"Fatal error: can't open the append log file for reading: %s",strerror(en));
             return AOF_OPEN_ERR;
         } else {
-            serverLog(LL_WARNING,"The append log file doesn't exist: %s",strerror(errno));
+            serverLog(LL_WARNING,"The append log file %s doesn't exist: %s",filename,strerror(errno));
```

Comment:
```suggestion
            serverLog(LL_WARNING,"Failed to open the append log file %s: %s",filename,strerror(errno));
```

The error message was incorrect, it may exist but fail to open for other reasons.

Reply:
The situation you mentioned has been handled in the if branch, and definitely doesn't exist error in the else branch (this code is currently redis used, I did not modify it)